python/tests/specifications/test_spec/plugin/specifications/specifications_generate_a_specification_csv.feature

```diff
@@ -0,0 +1,15 @@
+# WARNING - DO NOT EDIT - YOUR CHANGES WILL NOT BE PROTECTED.
+# This file is auto-generated by the aac gen-plugin and may be overwritten.
+
+
+Feature: Generate a Specification CSV
+
+  Scenario: Successful CSV Generation
+  
+      Given The {{spec-csv.input.architecture-file}} contains a valid architecture specification.
+      When The aac-spec app is run with the spec-csv command.
+      Then The aac-spec app is run with the spec-csv command.
+      
+    
+  
+  
```

Comment:
Not entirely sure what happened to get so many empty lines...can manually remove them and shouldn't be an issue. Also, making a note for the template feature to make sure when we work on the feature file template this gets evaluated.